tutorials/nnvm/from_tensorflow.py

```diff
@@ -76,7 +75,8 @@
     graph = tf.import_graph_def(graph_def, name='')
     # Call the utility to import the graph definition into default graph.
     graph_def = nnvm.testing.tf.ProcessGraphDefParam(graph_def)
-
+    # Add shapes to the graph.
+    graph_def = nnvm.testing.tf.AddShapesToGraphDef('softmax')
```

Comment:
I have a question. It seems that `AddShapesToGraphDef` override the graph_def returned from `ProcessGraphDefParam`. Is it no problem? `AddShapesToGraphDef` is only used for type checking?

Reply:
Thats not a problem as AddShapesToGraphDef take the default graph which is in tf session and is same as graph_def earlier. This step just re freezes the same graph with shapes added .

Reply:
Ok, I understand it. Thanks.